web/themes/custom/caesar/templates/patterns/atoms/button/a-button.ui_patterns.yml

```diff
@@ -47,3 +47,8 @@ a_button:
           - core/normalize
           - core/drupal
           - core/once
+          - a-button/choices
+    - choices:
+        js:
+        # TODO: review `installe-paths` option with `component-library` option
+          ../../../../../../../libraries/choices/src/scripts/choices.js: {}
```

Comment:
you will no be able to put correct path here, because path may vary from the way you are adding library. like with mereg plugin library mey be save in folder1 and with regular way `composer require` library may be saved in folder2, also project may easily declare its own library path, because of "security rules" requested by business for example

Reply:
Another important thing is that `choices` must be declared here in `dependencies` section like `- a-button/choices`

Reply:
https://github.com/skilld-labs/ui-patterns-kaizen/pull/6/files#diff-18b2c176645f84ad2364ee6fd476b7a1c3686f1d44c123a7a00a468e281afd20R53 
Added TODO here.